multi-agent/adapters/inbound/flask/endpoints/builtins.py

```
@@ -0,0 +1,432 @@
+
+# ─────────────────────────────────────────────────────────────────────────────
+#  Built-in resource endpoints
+# ─────────────────────────────────────────────────────────────────────────────
+
+@builtins_bp.route("/builtins.list", methods=["GET"])
+@require_admin_access
+@from_query({
+    "category": fields.Str(required=False),
+    "type": fields.Str(required=False),
+})
+def list_builtins(category=None, type=None):
```

Ruff (0.16.1)

[error] Function argument `type` is shadowing a Python builtin

(A002)